newstart/index.html

```diff
@@ -122,117 +123,38 @@ <h1 id="main-header" class="screenreader-only">Kids &amp; Teens Get a New Start
             </svg>
           </div>
           <span class="clearfix">Serving Bronx, Manhattan, and Staten Island library card holders</span>
-          <a href="https://www.nypl.org/newstart">Learn More</a>
+          <a href="https://www.nypl.org/backtoschool">Learn More</a>
 
         </div>
         <div class="library-logos__blocks logo-block column-one-third queens-logo">
           <div class="svg-container">
-            <svg version="1.0" xmlns="http://www.w3.org/2000/svg"
-            width="1114.000000pt" height="601.000000pt" viewBox="0 0 1114.000000 601.000000"
-            preserveAspectRatio="xMidYMid meet" aria-labelledby="queens-public-library-logo" class="svg-icon queens-icon">
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 348 56" preserveAspectRatio="xMidYMid meet" aria-labelledby="queens-public-library-logo" class="svg-icon queens-icon">
               <title id="queens-public-library-logo">Queens Public Library</title>
-              <metadata>
```

Comment:
This is the new SVG for the Queens logo. Did you delete this?

Reply:
ah oops yes I put the old one back, was basing off old code, can you put it back?